### PR TITLE
Grpc 1.3.1

### PR DESCRIPTION
**Pull request by KindDragon**

### CODE REVIEW

File: ports/c-ares/portfile.cmake [hunk 1]
```diff
@@ -0,0 +1,46 @@
+if(VCPKG_CRT_LINKAGE STREQUAL static)
+  message(FATAL_ERROR "C-Ares only supports dynamic crt linkage.")
```
Comment (codicodi) [inline ports/c-ares/portfile.cmake]: Any specific reason why static CRT is disabled? 
Note that [vcpkg_configure_cmake](https://github.com/Microsoft/vcpkg/blob/eb3d052a52a7c82bf4bcaed2eef8354ce8c0bf0d/scripts/cmake/vcpkg_configure_cmake.cmake#L80) sets up appropriate compiler flags automatically, so in most cases portfiles don't need to worry about CRT linkage at all (unless buildscript explicitly overrides them, but this doesn't seem to be the case).

File: ports/c-ares/CONTROL [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Source: c-ares
+Version: 1.12.0-1
```
Comment (codicodi) [inline ports/c-ares/CONTROL]: The version you're downloading here refer to is like ~50 commits after that release, so may want to reflect it here. The `ares_version.h` says 1.12.1-DEV so you may go with, for example, 1.12.1-dev-40eb41f.

File: ports/c-ares/portfile.cmake (hunk 1 above)
Comment (KindDragon) [inline ports/c-ares/portfile.cmake]: Ohh.. I don't know about that

Then correct me here too https://github.com/Microsoft/vcpkg/pull/1077#discussion_r115922959 :-)

**Comment by ras0219-msft** [comment]
A huge PR, and huge thanks 😊!

I verified grpc:x86-windows and protobuf:x64-windows as part of the merge.

**Pull request merged**